concepticondata/conceptlists/Healey-1970-364.tsv

```diff
@@ -0,0 +1,365 @@
+Healey-1970-364-76	76	do	813	DO
+Healey-1970-364-77	77	dog	2009	DOG
+Healey-1970-364-78	78	dream	1920	DREAM (SOMETHING)
+Healey-1970-364-79	79	drink	1401	DRINK
+Healey-1970-364-80	80	dry	1398	DRY
+Healey-1970-364-81	81	dumb	1492	DUMB
+Healey-1970-364-82	82	ear	1247	EAR
+Healey-1970-364-83	83	earth	1228	EARTH (SOIL)
+Healey-1970-364-84	84	earthquake	6	EARTHQUAKE
+Healey-1970-364-85	85	eat	1336	EAT
+Healey-1970-364-86	86	egg	744	EGG
+Healey-1970-364-87	87	elbow	981	ELBOW
+Healey-1970-364-88	88	empty	1624	EMPTY
+Healey-1970-364-89	89	enemy	1358	ENEMY
+Healey-1970-364-90	90	enough	1881	ENOUGH
+Healey-1970-364-91	91	excreta	47	EXCRETA
+Healey-1970-364-92	92	extinguish	127	EXTINGUISH
+Healey-1970-364-93	93	eye	1248	EYE
+Healey-1970-364-94	94	eyelashes	1540	EYELASH
+Healey-1970-364-95	95	far away	1406	FAR
+Healey-1970-364-96	96	farm	201	FARM
+Healey-1970-364-97	97	fat (n)	323	FAT (ORGANIC SUBSTANCE)
+Healey-1970-364-98	98	fault	134	FAULT
+Healey-1970-364-99	99	feather	1201	FEATHER
+Healey-1970-364-100	100	fell, to	355	CUT DOWN
+Healey-1970-364-101	101	female	2105	FEMALE PERSON
+Healey-1970-364-102	102	fence	1690	FENCE
+Healey-1970-364-103	103	fetch	3551	FETCH
+Healey-1970-364-104	104	find, see		
+Healey-1970-364-105	105	finger	1303	FINGER
+Healey-1970-364-106	106	little finger	2774	LITTLE FINGER
+Healey-1970-364-107	107	fire	221	FIRE
+Healey-1970-364-108	108	firewood	10	FIREWOOD
+Healey-1970-364-109	109	fish	227	FISH
+Healey-1970-364-110	110	flare		
+Healey-1970-364-111	111	flat	1633	FLAT
+Healey-1970-364-112	112	flea	232	FLEA
+Healey-1970-364-113	113	flesh (of body)	2259	FLESH
+Healey-1970-364-114	114	flower	239	FLOWER
+Healey-1970-364-115	115	fly, to	1441	FLY (MOVE THROUGH AIR)
+Healey-1970-364-116	116	flying fox	235	FLYING FOX
+Healey-1970-364-117	117	foot	1301	FOOT
+Healey-1970-364-118	118	four	1500	FOUR
+Healey-1970-364-119	119	frog	503	FROG
+Healey-1970-364-120	120	fruit	1507	FRUIT
+Healey-1970-364-121	121	garden	586	GARDEN
+Healey-1970-364-122	122	gather	7	GATHER
+Healey-1970-364-123	123	get up	570	STAND UP
+Healey-1970-364-124	124	ghost	1175	GHOST
+Healey-1970-364-125	125	girl	1646	GIRL
+Healey-1970-364-126	126	give	1447	GIVE
+Healey-1970-364-127	127	go away	1752	LEAVE
+Healey-1970-364-128	128	goes out, (fire)	127	EXTINGUISH
+Healey-1970-364-129	129	grandchild	1617	GRANDCHILD
+Healey-1970-364-130	130	grandparent	2399	GRANDPARENT
+Healey-1970-364-131	131	ground	2159	GROUND
+Healey-1970-364-132	132	guard	1940	GUARD
+Healey-1970-364-133	133	hair (of head, of body)	1040	HAIR
+Healey-1970-364-134	134	hand	1277	HAND
+Healey-1970-364-135	135	he	1211	HE
+Healey-1970-364-136	136	head	1256	HEAD
+Healey-1970-364-137	137	hear	1408	HEAR
+Healey-1970-364-138	138	heart	1223	HEART
+Healey-1970-364-139	139	heavy	1210	HEAVY
+Healey-1970-364-140	140	heel	980	HEEL
+Healey-1970-364-141	141	her	2474	HER (GENITIVE OF SHE)
+Healey-1970-364-142	142	hide (it)	602	HIDE (CONCEAL)
+Healey-1970-364-143	143	high	1265	HIGH
+Healey-1970-364-144	144	his	2092	HIS OR HER
```

Comment:
Could we map this to the narrow concept 2473	HIS (GENITIVE OF HE)?

Reply:
ok